frontend/src/components/common/text-input/styles.module.scss

```diff
@@ -44,6 +51,10 @@
   color: var(--color-error);
 }
 
+.inputBlock > span {
+  color: var(--color-error);
+}
```

Comment:
Label should be red. See markup
![image](https://user-images.githubusercontent.com/60396171/110824064-5f93de00-829b-11eb-8d82-3dbb910433db.png)